app/src/main/AndroidManifest.xml

```diff
@@ -33,21 +33,37 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme.NoActionBar">
+
+        <!--        new refactor-->
+        <activity
+            android:name=".onboarding.ui.OnboardingActivity"
+            android:exported="true"
+            android:launchMode="singleTop"
+            android:screenOrientation="portrait">
+<!--            <intent-filter>-->
+<!--                <action android:name="android.intent.action.MAIN" />-->
+<!--                <category android:name="android.intent.category.LAUNCHER" />-->
+<!--            </intent-filter>-->
+        </activity>
+
+        <!--        old code-->
         <activity
             android:name=".ui.MainActivity"
-            android:screenOrientation="portrait"
             android:alwaysRetainTaskState="true"
+            android:exported="true"
             android:launchMode="singleTop"
-            android:theme="@style/AppTheme.NoActionBar.Launcher"
-            android:exported="true">
+            android:screenOrientation="portrait"
+            android:theme="@style/AppTheme.NoActionBar.Launcher">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
             <intent-filter>
                 <action android:name="android.intent.action.VIEW" />
+
```

Comment:
why these extra lines?

Reply:
that is the most common formatting style in Android. This is for better visibility between Actions/Categories etc.